pallets/staking/fuzzer/Cargo.lock

```diff
@@ -0,0 +1,2294 @@
+# This file is automatically @generated by Cargo.
```

Comment:
`Cargo.lock` inside a pallet? Only root workspace should have a `Cargo.lock` file.

Reply:
Nice catch @fmiguelgarcia I forgot about this